.github/workflows/build.yaml

```diff
@@ -43,15 +43,18 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v2
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
The latest version of this GitHub action is [v3](https://github.com/docker/setup-qemu-action/releases/tag/v3.0.0). Do you prefer to upgrade from v2?

Reply:
Upgrade to v3 will be done in separate PR and will be done in all github actions.